src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ExpressRouteCircuitTests.ps1

```diff
@@ -42,9 +42,7 @@ function Test-ExpressRouteRouteFilters
       $resourceGroup = New-AzureRmResourceGroup -Name $rgname -Location $location
 
       # Create the route filter
-      $job = New-AzureRmRouteFilter -Name $filterName -ResourceGroupName $rgname -Location $location -Force -AsJob
```

Comment:
Why are you removing this?

Reply:
Did not intend to change this. Not sure how these diffs got introduced. Resolved